interface/billing/edit_payment.php

```diff
@@ -455,669 +489,695 @@
-      return false;
-     }
-    if(!CheckPayingEntityAndDistributionPostFor())//Ensures that Insurance payment is distributed under Ins1,Ins2,Ins3 and Patient paymentat under Pat.
-     {
-      return false;
-     }
-    if(CompletlyBlankBelow())//The newly added distribution rows are checked.
-     {
-      alert("<?php echo htmlspecialchars(xl('Fill any of the Below Row.'), ENT_QUOTES) ?>")
-      return false;
-     }
-    PostValue=CheckUnappliedAmount();//Decides TdUnappliedAmount >0, or <0 or =0
-    if(PostValue==1)
-     {
-      alert("<?php echo htmlspecialchars(xl('Cannot Modify Payments.Undistributed is Negative.'), ENT_QUOTES) ?>")
-      return false;
-     }
-    if(PostValue==2)
-     {
-        if(confirm("<?php echo htmlspecialchars(xl('Would you like to Modify and Finish Payments?'), ENT_QUOTES) ?>"))
-         {
-            UnappliedAmount=document.getElementById('TdUnappliedAmount').innerHTML*1;
-            if(confirm("<?php echo htmlspecialchars(xl('Undistributed is'), ENT_QUOTES) ?>" + ' ' + UnappliedAmount + '.' + "<?php echo htmlspecialchars('\n');echo htmlspecialchars(xl('Would you like the balance amount to apply to Global Account?'), ENT_QUOTES) ?>"))
-             {
-                document.getElementById('mode').value='FinishPayments';
-                document.getElementById('global_amount').value='yes';
-                top.restoreSession();
-                document.forms[0].submit();
-             }
-            else
-             {
-                document.getElementById('mode').value='FinishPayments';
-                top.restoreSession();
-                document.forms[0].submit();
-             }
-         }
-        else
+    <?php Header::setupHeader(['datetime-picker']);?>
+    <!--<?php // (function_exists('html_header_show')) html_header_show(); ?>
+    <link href="<?php //echo $css_header;?>" rel="stylesheet" type="text/css">-->
+    <!--<link href="../../library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" rel="stylesheet" type="text/css">-->
+    <!--<link href="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker-2-5-4/build/jquery.datetimepicker.min.css" rel="stylesheet"><!-- supporting javascript code -->
+
+    <!--<script src="../../library/textformat.js?v=<?php echo $v_js_includes; ?>" type="text/javascript">
+    </script>-->
+    <script src="../../library/dialog.js?v=<?php echo $v_js_includes; ?>" type="text/javascript">
+    </script>
+    <!--<script src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-7-2/index.js" type="text/javascript">
+    </script>-->
+    <!--<script src="../../library/js/fancybox/jquery.fancybox-1.2.6.js" type="text/javascript">-->
+    </script>
+    <!--<script src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker-2-5-4/build/jquery.datetimepicker.full.min.js" type="text/javascript">
+    </script>-->
```

Comment:
remove above commented out code.
also remove dialog.js line since this i automatically included in the setupHeader call